gatsby/src/styles/global.scss

```diff
@@ -0,0 +1,18 @@
+@import './reset';
+
+html {
+  scroll-behavior: smooth;
+}
+
+body {
+  color: #666;
+  font-family: -apple-system, BlinkMacSystemFont, Segoe UI, Roboto, Helvetica Neue, Ubuntu, sans-serif;
```

Comment:
Maybe we should start with Roboto so that it's in line with all our other sites?  And Montserrat for the `hx` elements

Reply:
I went with this stack because using the system font's first resulted in a noticeably crisper font. Twitter Bootstrap 4 does something similar - https://github.com/twbs/bootstrap/blob/9f9e4d04a127b3505f01bb6c341d3d40430f39ff/scss/_variables.scss#L386.